src/utils/polling.js

```diff
@@ -114,16 +115,49 @@ async function pollSubscriptionTopics() {
         };
         db.upsert('polling', poll, (p) => p.resource === resourceToPoll);
 
+        const newResources = [];
+        const modifiedResources = [];
+
         // Store fetched resources in local database
         if (data.total > 0) {
           logger.info(`Storing ${data.total} fetched resource(s) for ${resourceToPoll} into database.`);
           const resources = data.entry.map(entry => entry.resource);
 
           resources.forEach((resource) => {
             const collection = `${resource.resourceType.toLowerCase()}s`;
+
+            // Determine if resource is new before inserting
+            const storedResource = db.select(collection, r => r.id === resource.id);
+            if (storedResource.length === 0) newResources.push(resource);
+            else modifiedResources.push(resource);
+
             db.insert(collection, resource);
```

Comment:
It would be great if there was a way to identify new/updated without having to store (ie, duplicate) every resource in our own internal DB. Not sure we need a solution immediately but something to think about.

One quick idea - could we store just the ID/fullUrl and use that to determine whether something is new/updated? We don't actually compare the content right?

Reply:
Yea that's true I can change it to just store the id. I think when I first implemented the polling I thought we would have to compare the contents ourselves but since we're just using `lastUpdated` and letting the FHIR server do the work we don't need to have the full resource.